include/libboostasio.h

```diff
@@ -27,33 +26,35 @@
 #include <boost/bind.hpp>
 
 
-///////////////////////////////////////////////////////////////////
```

Comment:
I'd wrap this in an anonymous namespace.

Reply:
True, that I will add.